src/bandada-sdk.ts

```diff
@@ -1,33 +1,53 @@
 import { ApiSdk } from "@bandada/api-sdk"
+import figlet from "figlet"
+import chalk from "chalk"
+import * as emoji from "node-emoji"
 import "dotenv/config"
 
+// Theme.
+const log = console.log
```

Comment:
```suggestion
const { log } = console
```

There was an ESLint error that can be fixed with this.